_security-plugin/access-control/document-level-security.md

```diff
@@ -145,5 +145,5 @@ plugins.security.dls.mode: filter-level
 | Evaluation mode | Parameter | Description | Usage |
 :--- | :--- | :--- | :--- |
 Lucene-level DLS | `lucene-level` | This setting makes all DLS queries apply to the Lucene level. | Lucene-level DLS modifies Lucene queries and data structures directly. This is the most efficient mode but does not allow certain advanced constructs in DLS queries, including TLQs.
-Filter-level DLS | `filter-level` | This setting makes all DLS queries apply to the filter level. | In this mode, OpenSearch applies DLS by modifying queries that OpenSearch receives. This allows for term-level lookup queries in DLS queries, but you can only use the `get`, `search`, `mget`, and `msearch` operations to retrieve data from the protected index. Additionally, cross-cluster searches are limited with this mode.
-Adaptive | `adaptive-level` | The default setting that allows OpenSearch to automatically choose the mode. | DLS queries without TLQs are executed in Lucene-level mode, while DLS queries that contain TLQ are executed in filter- level mode.
+Filter-level DLS | `filter-level` | This setting makes all DLS queries apply to the filter level. | In this mode, OpenSearch applies DLS by modifying queries that OpenSearch receives. This allows for TLQs in DLS queries, but you can only use the `get`, `search`, `mget`, and `msearch` operations to retrieve data from the protected index. Additionally, cross-cluster searches are limited with this mode.
```

Comment:
I would avoid the repetition of "OpenSearch applies DLS by modifying queries that OpenSearch receives." How about something like "In this mode, OpenSearch applies DLS by modifying the queries received"? Also, the descriptions should ideally use the same structure. I would change the second description to "By default, this setting allows OpenSearch to automatically choose the mode."